<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fixed_Value_Setting</class>
 <widget class="QWidget" name="Fixed_Value_Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>155</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_time">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>310</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>定点功能</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1280</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="QWidget" name="form" native="true">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>78</y>
      <width>954</width>
      <height>494</height>
     </rect>
    </property>
    <widget class="QLabel" name="program_label">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>240</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>程    式</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="temperature_label">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>72</y>
       <width>240</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>温      度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="humidity_label">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>142</y>
       <width>240</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>湿      度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="slope_time_label">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>212</y>
       <width>240</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>斜率时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="constant_temperature_label">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>282</y>
       <width>240</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>恒温时间</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sequence_label">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>352</y>
       <width>240</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>时      序</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="wait_label">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>422</y>
       <width>240</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>W a i t</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="t1">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>2</y>
       <width>710</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ONE-POINT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="t2">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>72</y>
       <width>710</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="t3">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>142</y>
       <width>710</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="t4">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>212</y>
       <width>710</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>H               M               S</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="t5">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>282</y>
       <width>710</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>H               M               S</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="t6">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>352</y>
       <width>710</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TS1:          TS2:          TS3:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="t7">
     <property name="geometry">
      <rect>
       <x>242</x>
       <y>422</y>
       <width>710</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>[0=N/1=T/2=H]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="border_left">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>1</width>
       <height>492</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="border_right">
     <property name="geometry">
      <rect>
       <x>951</x>
       <y>2</y>
       <width>1</width>
       <height>492</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="border_top">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>952</width>
       <height>1</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="border_buttom">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>491</y>
       <width>952</width>
       <height>1</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="temperature_lineEdit">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>85</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="humidity_lineEdit">
     <property name="geometry">
      <rect>
       <x>455</x>
       <y>155</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="slopeTime_H_lineEdit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>225</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="slopeTime_M_lineEdit">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>225</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="slopeTime_S_lineEdit">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>225</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="constantTemp_H_lineEdit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>295</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="constantTemp_M_lineEdit">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>295</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="constantTemp_S_lineEdit">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>295</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="TS1_lineEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>365</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="TS2_lineEdit">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>365</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="TS3_lineEdit">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>365</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="Wait_lineEdit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>435</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="Footer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="font_page_pbtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目  录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saving_pbtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保  存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_page_pbtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>3</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="status_box" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="status_box_line">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>1</y>
       <width>2</width>
       <height>59</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_text">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>定值运行</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="status_1">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>15</y>
       <width>25</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>15</y>
       <width>22</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>15</y>
       <width>15</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_4">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>18</y>
       <width>30</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>16</y>
       <width>27</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>15</y>
       <width>30</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
